src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -1777,8 +1777,6 @@ private void AppendSymbolsWithName(ArrayBuilder<ISymbol> results, string name, B
 
         private Symbol RemapSymbolIfNecessary(Symbol symbol)
         {
-            if (!Compilation.IsNullableAnalysisEnabled) return symbol;
```

Comment:
Why did we remove this check? I tried digging into `RemapSymbolIfNecessaryCore` to see if it was pushed down to those methods but didn't see it.

Reply:
The check is now more specific and is handled indirectly in `MemberSemanticModel.RemapSymbolIfNecessaryCore()` in the call to `EnsureNullabilityAnalysisPerformedIfNecessary()`.

---
In reply to: [551576200](https://github.com/dotnet/roslyn/pull/49953#discussion_r551576200) [](ancestors = 551576200)